src/pudl/convert/epacems_to_parquet.py

```diff
@@ -91,27 +93,47 @@ def parse_command_line(argv):
     return arguments
 
 
-def get_epacems_job():
-    """Create an epacems_job wrapped by to be wrapped by reconstructable."""
-    return Definitions(
-        assets=pudl.etl.default_assets,
-        resources=pudl.etl.default_resources,
-        jobs=[define_asset_job("epacems_job", selection="hourly_emissions_epacems")],
-    ).get_job_def("epacems_job")
+def epacems_job_factory(loglevel: str, logfile: str) -> Callable[[], JobDefinition]:
+    """Factory for parameterizing a reconstructable epacems job.
+
+    Args:
+        loglevel: The log level for the job's execution.
+        logfile: Path to a log file for the job's execution.
+
+    Returns:
+        The job definition to be executed.
+    """
+
+    def get_epacems_job():
+        """Create an epacems_job wrapped by to be wrapped by reconstructable."""
+        pudl.logging_helpers.configure_root_logger(logfile=logfile, loglevel=loglevel)
```

Comment:
I would expect to be able to configure the root logger just in the CLI script, but I guess that doesn't work and this does?

Reply:
Yeah, I'm not sure why this works tbh :/ Maybe because dagster is spawning different processes that use the default logging configuration?